buildchow/todo.adoc

```diff
@@ -0,0 +1,69 @@
+= Advantages noticed within a week:
+
+* fixing JRE across all jobs
+* relasing target platform without breaking builds
+* using 9.0 instead of 4.3.mars for stream
```

Comment:
this was when both you and michael within a week messed up the setup of new jobs since you mistyped or had a bad regex that caused 9.0 / 4.3.mars mixup.

Much less chance of that happening with the templates; and if it does its a one line change instead of ~22 or so in the current model.